nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -439,6 +439,13 @@ in
 
   config = {
 
+    assertions = [
+      {
+        assertion = (!config.nix.distributedBuilds) && config.nix.buildMachines != [];
```

Comment:
Since this uses `&&`, it requires both sides to be true for the assertion to succeed, and `config.nix.buildMachines` is `[]` if you haven't set up distributed builds.

Reply:
oops - thanks, i was thinking of it backwards; fixed in67b6e5639126ab6e8ec5d2b303aa9736a626b609